src/test/unittest/unittest.sh

```diff
@@ -1623,11 +1627,24 @@ function require_valgrind_tool() {
 		exit 0
 	fi
 
-	if [ "$tool" == "pmemcheck" -o "$tool" == "helgrind" ]; then
+	if [ "$tool" == "helgrind" ]; then
 		valgrind --tool=$tool --help 2>&1 | \
 		grep -qi "$tool is Copyright (c)" && true
 		if [ $? -ne 0 ]; then
-			msg "$UNITTEST_NAME: SKIP valgrind with $tool required"
+			msg "$UNITTEST_NAME: SKIP Valgrind with $tool required"
+			exit 0;
+		fi
+	fi
+	if [ "$tool" == "pmemcheck" ]; then
+		out=`valgrind --tool=$tool --help 2>&1`
+		echo "$out" | grep -qi "$tool is Copyright (c)" && true
+		if [ $? -ne 0 ]; then
+			msg "$UNITTEST_NAME: SKIP Valgrind with $tool required"
+			exit 0;
+		fi
+		echo "$out" | grep -qi "expect-fence-after-clflush" && true
+		if [ $? -ne 0 ]; then
+			msg "$UNITTEST_NAME: SKIP pmemcheck does not support --expect-fence-after-clflush option. Please update it."
```

Comment:
Can we indicate the minimal version or something like that?
*update it* is very general.

Reply:
Yes, I know, but I don't know how can I improve this message, because pmemcheck does not have any versioning scheme :(.